dashboard/app/assets/stylesheets/application.scss

```diff
@@ -74,8 +74,11 @@ body.oceans-blue {
   background-color: $oceans_deep_blue;
 }
 
-body.music-black {
+body.music-black, body.background-dark {
```

Comment:
Can we remove the `music-black` class or is that still needed for older levels?

Reply:
Older levels rely on this.  We can set the background for all of the music lessons on levelbuilder after this PR is available, and then we can remove this.